### DISCUSSION TITLE
Add chin line (front view)

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#114] [Q&A] [answered]
![chin line](https://user-images.githubusercontent.com/36077492/129901297-6dcb4b2e-fd83-4f9e-9107-b2d6e5ee1901.png)

How can I add a chin line in the front view. (Like the red line I draw)

**Comment by HSDSZ** [comment]
decreasing the line normal threashold wont work. Because decrasing it will create a lot of unwanted line on the face

**Comment by pragma37** [comment, marked as answer]
You could use a texture or vertex colors to scale the line normal/depth threshold only on the chin.

That said, I think this would be better solved through modeling rather than shading.
https://www.slideshare.net/ASW_Yokohama/guilty-gear-xrdtips-124324946/27

**Reply by HSDSZ** [reply]
![2](https://user-images.githubusercontent.com/36077492/129944511-15faceea-a0f3-476c-998c-44dd520c16c5.PNG)
Thanks, after retopo, the result is better
